changelog.d/20763_loki_sink_drop_invalid_timestamp_event.fix.md

```diff
@@ -0,0 +1,3 @@
+Loki sink drop events with timestamp can not parsed by chrono.
```

Comment:
What about something like this?

```suggestion
Loki sink now drops events with non-parseable timestamps.
```

Reply:
Get it, but the word `parseable ` is not recognized. Maybe should update the checklist.
![image](https://github.com/user-attachments/assets/8b28ffba-b7ad-4810-a3d7-81feaf2f4bdd)

Reply:
i'm not an english native, maybe @jszwedko can help us here😁. I think the spelling is correct